packages/node_modules/@webex/plugin-meetings/src/locus-info/controlsUtils.js

```diff
@@ -47,11 +47,11 @@ ControlsUtils.getControls = (oldControls, newControls) => {
     previous,
     current,
     updates: {
-      hasRecordingPausedChanged: !isEqual(
+      hasRecordingPausedChanged: current.record && !isEqual(
```

Comment:
Why do you need to do this check? If it was paused (`previous.record.paused === true`) and now there isn't a `current.record`, wouldn't that be a `recording pause` change?

Reply:
If you need to have the check, perhaps the following change:
```suggestion
      hasRecordingPausedChanged: hasRecord(current) && !isEqual(
```

Reply:
yeah I'm not following this either. @arun3528 let's talk tomorrow about this.

Reply:
When it comes to meeting end or removed the last event which comes in has controls(current) but it does not have record. on it so had to add that check

Reply:
right we are checking that before the comparison
```js
hasRecord(current) && current.record.recording
```

Reply:
sorry i meant to say current.record is not found

Reply:
can we do this instead then? or something similar to this?
```js
 const hasRecord = (controls) => controls && typeof controls.record !== 'undefined';
```

I'm just trying to have the check logic together instead of having it there.

Reply:
actually this works too:

```js
const hasRecord = (controls) => controls && !!controls.record;
```